tests/phpunit/includes/Core/Util/Google_URL_Matcher.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Google_URL_Matcher
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Core\Util\Google_URL_Matcher_Trait;
+
+/**
+ * Class Google_URL_Matcher
+ *
+ * A proxy class just to access protected trait methods.
```

Comment:
Not sure how I feel about adding a class just for testing (that isn't under `tests/`).

I would think we should move this together with other test sources as other classes will use the trait.

Reply:
Not sure what you mean, this class _is_ under `tests/`. :)

Reply:
🤦‍♂